src/include/daos/common.h

```diff
@@ -794,6 +794,7 @@ enum {
 /** interoperability failure inject */
 #define FLC_SMD_DF_VER			(DAOS_FAIL_UNIT_TEST_GROUP_LOC | 0x70)
 #define FLC_POOL_DF_VER			(DAOS_FAIL_UNIT_TEST_GROUP_LOC | 0x71)
+#define DAOS_POOL_UPGRADE_CONT_ABORT	(DAOS_FAIL_UNIT_TEST_GROUP_LOC | 0x73)
```

Comment:
nit, seems to be sorted according to their values => definition of DAOS_POOL_UPGRADE_CONT_ABORT should probabyl come after DAOS_FAIL_LOST_REQ ?